### ISSUE TITLE
function-comma-space-after flags multi-line attibute

### ISSUE THREAD

**Issue by crystalfp** [stylelint/stylelint#6277]
### What steps are needed to reproduce the bug?

I have the following formatted as stylelint asks for multi-line property:
```
.tiles-layout-ghost-class {
  background:
    linear-gradient(
      to bottom,
      rgb(0, 174, 255) 0%,
      rgb(1, 45, 95) 35%,
      rgb(1, 45, 95) 65%,
      rgb(0, 174, 255) 100%
    );
}
```
the remaining error is: `Expected single space after ","  function-comma-space-after` that I cannot silence with the `function-comma-space-after: always` rule. This rule is needed and not set to `never` because in single line functions I want a space after the comma like in `translate(3px, 5px)`

### What Stylelint configuration is needed to reproduce the bug?

```yaml

extends:
  - stylelint-config-standard-vue
  - stylelint-config-recommended-vue/scss

plugins:
  - stylelint-scss

customSyntax: postcss-html

rules:

  declaration-block-semicolon-newline-after: always-multi-line
  declaration-colon-newline-after: always-multi-line
  declaration-colon-space-after: always-single-line
  declaration-colon-space-before: never
  function-comma-space-after: always
```

### How did you run Stylelint?

CLI `stylelint ./src/App.vue` with `.stylelintrc.yaml`

### Which version of Stylelint are you using?

`14.10.0`

### What did you expect to happen?

I don't want to receive the error with multiline functions.

### What actually happened?

```
 181:16  ✖  Expected single space after ","  function-comma-space-after
 182:26  ✖  Expected single space after ","  function-comma-space-after
 183:25  ✖  Expected single space after ","  function-comma-space-after
 184:25  ✖  Expected single space after ","  function-comma-space-after
```

### Does the bug relate to non-standard syntax?

No, standard scss that follows the stylelint rules for multiline

### Proposal to fix the bug

Add to `function-comma-space-after` rule, besides `"always"|"never"`, an `"always-single-line"` option.

**Comment by ybiquitous** [comment]
@crystalfp Thanks for using the template. We can reproduce the errors in the [demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIB0AXAlgGxgZwLQYEMBPCAVxVwHMALCbCsQ7bAAmAB0A7FlgIwLABrSgCcynKPC48eGNJxgERVEQShoYnFAAppMnigh8IKQwFsANHv0jKvbQAYLLAIwB2ACzOATAFZfAJQsDgCkVtz6LLb2Ls4evs4AnIEsAMy+YdYy0dqxLPFJKQBsGeGRUXaOzu5eLH4pLg6hWQEA3FwAvlwgFiDYRFoEAB4I4Mw94BCcAGZolKMcEewgIqRY2MvwbFnL06ScYOhTuJBmZgS42AAOAjC4BNMoMCKbLMsEGADuxBsgel2cDogDpAA).

Unfortunately, stylistic rules like `function-comma-space-after` [will be frozen and no longer fixed](https://stylelint.io/user-guide/rules/list#stylistic-issues). Please consider using a pretty printer like Prettier, or turning off the rule via your config file or `/* stylelint-disable */` comments.
